storage-proofs/src/piece_inclusion_proof.rs

```diff
@@ -188,13 +265,17 @@ impl<H: Hasher> PieceInclusionProof<H> {
     /// position encoded in the proof.
     /// `piece_leaves` and `sector_leaves` are in units of `Domain` (i.e. `NODE_SIZE` = 32 bytes).
 
-    fn verify(
+    pub fn verify(
         &self,
         root: &H::Domain,
         comm_p: &H::Domain,
         piece_leaves: usize,
         sector_leaves: usize,
     ) -> bool {
+        if sector_leaves == 0 {
```

Comment:
QUESTION: Why does this guard exist? Under what circumstances would `sector_leaves` be equal to `0`?